contractcourt/contract_resolvers.go

```diff
@@ -61,6 +64,18 @@ type ContractResolver interface {
 	Stop()
 }
 
+// htlcContractResolver is the required interface for htlc resolvers.
+type htlcContractResolver interface {
+	ContractResolver
+
+	// HtlcPoint returns the htlc's outpoint on the commitment tx.
+	HtlcPoint() wire.OutPoint
+
+	// Supplement adds additional information to the resolver that is
+	// required before Resolve() is called.
+	Supplement(htlc *channeldb.HTLC) error
```

Comment:
atm none of these can fail (other than panicking), does it need a return value? are there cases in the future where we might that would justify leaving it?

Reply:
I don't know any future cases, so removed it for now.